### DISCUSSION TITLE
Anxiety, OpenBSD break, COVID-19 and resuming work · poolp.org

### DISCUSSION THREAD

**Discussion by poolpOrg** [poolpOrg/poolp.org#118] [Articles]

**Comment by dataondeck** [comment]
I appreciate the level of personal honesty you bring to your writing. In the very cluttered space of technical blogs yours is one that stands out to me for this reason. 

Also I would like to thank you. As an email admin for a small ISP I did not find too much pride in my email related work although I found the underlying protocols interesting. I did not have much to offer in response to people who questioned why we don't just offload it to one of the big boys and move on. 

After reading your posts about the state of mail hosting, in which you described the importance of maintaining the decentralized nature of a communication protocol like SMTP, I felt a spark and began to take some pride in my small role in providing some decentralization. 

This spark has lit a fire and now I am seeking to make our little email service the best it can possibly be and also pursuing mail related projects in my free time which has provided me some natural direction in my technical journey. I have you to thank for this!

**Comment by ngortheone** [comment]
Hey Gilles, glad to have you back!

Your update reminded me of  [Drew's post](https://drewdevault.com/2020/01/21/Stress-and-happiness.html) from not so long ago.

We often forget the importance of rest, both physical and mental and we only notice when we push ourselves over the edge, driven by the sense of duty, perfection or something else. One of my colleagues once said "_It is software, it is never finished_" and it is the very truth about software. So no matter how hard you work - there will always be more work, more bugs, more features etc.

I am now using OpenSMTPD on my mail server and I find it to be much easier to comprehend than other alternatives so thank you for your hard work!

**Comment by poolpOrg** [comment]
Thanks @dataondeck !

Thanks @ngortheone , will read Drew's post tomorrow :-)

The main issue for me is being unable to decode that I'm stressed or anxious. The perfect example is these days with the confinement: I've been inside for ~3 weeks now, only went out 10 minutes once to grab food. I was asked by a friend yesterday if confinement was ok and I said yes because I don't feel it is stressful or anything, then a few hours later it struck me that I've been itching my right arm to blood for the last 3 weeks ever since I've confined myself. The connection between physical symptoms is just not there and this is why I need to put some kind of schedule to not push myself too far: I'm not able to properly assess it myself.

**Comment by ngortheone** [comment]
> I need to put some kind of schedule to not push myself 

This is how it works for me too:
- Set aside some time for exercise (20-30) minutes every day
- Set aside time for family
- Set aside some time to be alone, without electronic devices to reflect, introspect, think..
- Walk my dogs 2 times a day
- Do not use electronic devices(TV included) while eating/in bed/with family
- Go to bed after midnight

Especially exercise and walks help to clean my head from work and start reflecting.

Discipline is Freedom (C)

**Comment by NicolasCARPi** [comment]
Take care!

Best,
~Nico

**Comment by edmundlod** [comment]
Take care, Gilles! Thanks very much for all you do. I wish you and your family all the best and happiness.

**Comment by blakedot** [comment]
Bon courage & reposes-toi bien Gilles!
Merci encore pour OpenSMTPd & tes blogs.

**Comment by ghost** [comment]
Hi Gilles, while it may sound a bit weird coming from a total stranger, I just wanted to thank you for all the work you've done over the years with OpenBSD and OpenSMTPD. Your posts inspired me to ditch a BigMailerCorp and run my own mail server a while back and it has been both a great learning experience and also a freeing one. I sincerely wish you and your family the very best.

**Comment by poolpOrg** [comment]
Thanks @lquidfire @blakedot and @jsfierro <3
